app/controllers/manage_submissions_controller.rb

```diff
@@ -5,4 +5,9 @@ class ManageSubmissionsController < ApplicationController
   def index
     @challenges = current_user.challenge_manager_challenges
   end
+
+  def by_challenge_phase
+    @phase = Phase.find(params[:phase_id])
```

Comment:
Challenge managers can only access their own challenges. This query should be scoped to the current_user like the index route. This should work:
```suggestion
    @phase = current_user.challenge_manager_challenges.phases.find(params[:phase_id])
```